MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetCache.cs

```diff
@@ -38,6 +38,7 @@ public CacheEntry(Guid id, Guid containerId, AssetSource source, Object asset, C
 
 		private readonly List<CacheEntry> cache = new List<CacheEntry>(50);
 		private readonly Dictionary<Guid, List<CacheCallback>> cacheCallbacks = new Dictionary<Guid, List<CacheCallback>>(50);
+		private readonly Dictionary<Guid, Tuple<GLTFSceneImporter, GLTF.Schema.GLTFRoot>> importerCache = new Dictionary<Guid, Tuple<GLTFSceneImporter, GLTF.Schema.GLTFRoot>>(50);
```

Comment:
There's no point holding onto the importers after we've pulled all the assets out. Better to dispose of these immediately and free up the memory, then if we get a request to load this asset again, simply return the assets generated last time. So maybe holding onto URL to Guid[] would be better.